thrift/admin.thrift

```diff
@@ -359,6 +364,12 @@ struct RestoreDynamicConfigRequest {
   20: optional list<config.DynamicConfigFilter> filters
 }
 
+struct AdminDeleteWorkflowRequest {
+  10: optional string                       domain
+  20: optional shared.WorkflowExecution     execution
+  30: optional bool                         skipErrors
```

Comment:
What's `skipErrors` for? Perhaps rename it to `force` if that's inline with the intent.

Reply:
I have been using the same parameter names from admin local delete command. Since that name is also exposed to API, it's probably better to use the same name.